pandas/core/generic.py

```diff
@@ -3116,32 +3119,45 @@ class      (index) object 'bird' 'bird' 'mammal' 'mammal'
     def to_latex(
         self,
         buf=None,
+        *,
+        hrules=False,
         columns=None,
-        col_space=None,
-        header=True,
-        index=True,
-        na_rep="NaN",
-        formatters=None,
-        float_format=None,
-        sparsify=None,
-        index_names=True,
-        bold_rows=False,
         column_format=None,
-        longtable=None,
+        position=None,
```

Comment:
do we *need* to add all of these options? e.g. since we can already do this with styler